### PR TITLE
LPS-193462

### PR DESCRIPTION
**Pull request by TamyrisTorres**
Ticket: [LPS-193462](https://liferay.atlassian.net/browse/LPS-193462)

### CODE REVIEW

**Comment by TamyrisTorres** [comment]
ci:test:sf

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/BaseUpgradeCheck.java [hunk 1]
```diff
@@ -6,13 +6,36 @@
...
 import com.liferay.source.formatter.check.util.JavaSourceUtil;
 
+import java.util.List;
+
 /**
  * @author Nícolas Moura
  */
 public abstract class BaseUpgradeCheck extends BaseFileCheck {
 
+	public boolean validateParameters(
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/BaseUpgradeCheck.java:19]: @TamyrisTorres, let's rename to `hasValidParameters` instead to keep consistency with the recent methods created by us, for example `hasParameterTypes`.

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaAddAddressMethodCheck.java [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+					!hasClassOrVariableName(
+						"AddressService", content, content, methodCall)) {
+
+					continue;
+				}
+
+				String message = StringBundler.concat(
+					"Unable to format method addAddress from ",
+					"AddressLocalServiceUtil, AddressServiceUtil, ",
+					"AddressLocalService and AddressService. Fill the new ",
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaAddAddressMethodCheck.java:64]: @TamyrisTorres, can you sort alphabetically?

AddressLocalService, AddressLocalServiceUtil, AddressService, AddressServiceUtil

File: modules/util/source-formatter/src/main/resources/sourcechecks.xml [hunk 3]
```diff
@@ -1631,6 +1631,10 @@
 			<description name="Replaces with `compileInclude` the configuration attribute for dependencies in `build.gradle` that are listed at `Include-Resource` property at `bnd.bnd` associated file." />
 			<weight>2</weight>
 		</check>
+		<check name="UpgradeJavaAddAddressMethodCheck">
+			<category name="Upgrade" />
+			<description name="Fill the new parameter of the method 'addAddress' of AddressLocalServiceUtil, AddressServiceUtil, AddressLocalService and AddressService" />
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/resources/sourcechecks.xml:1636]: @TamyrisTorres, can you sort alphabetically?

AddressLocalService, AddressLocalServiceUtil, AddressService, AddressServiceUtil

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaAddAddressMethodCheck.testjava [hunk 4]
```diff
@@ -7,72 +7,41 @@ package com.liferay.source.formatter.dependencies.upgrade;
...
-			null, userId, className, classPK, null, null, street1, street2, street3, city, zip, regionId, countryId, typeId, mailing, primary, null, serviceContext);
-
-		// AddressService
-
-		_addressService.addAddress(
-			null, userId, className, classPK, null, null, street1, street2, street3, city, zip, regionId, countryId, typeId, mailing, primary, null, serviceContext);
-
-		// Test scenario when it is unable to validate parameter types
 
 		AddressLocalServiceUtil.addAddress(
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaAddAddressMethodCheck.testjava:46]: @TamyrisTorres, can you sort the service calls alphabetically?

**Comment by TamyrisTorres** [comment]
ci:test:sf

**Comment by TamyrisTorres** [comment]
ci:test:relevant

**Comment by TamyrisTorres** [comment]
ci:test:sf

**Comment by TamyrisTorres** [comment]
ci:test:relevant

**Comment by TamyrisTorres** [comment]
ci:forward

**Comment by TamyrisTorres** [comment]
ci:forward

**Comment by nicolas-sss** [comment]
ci:forward:force

**Pull request closed without merging**